release-notes/DxMs/DataAPI_change_log.md

```diff
@@ -7,6 +7,59 @@ uid: DataAPI_change_log
 > [!IMPORTANT]
 > At present, the Data API feature is only available in preview, if the soft-launch option *DataAPI* is enabled. For more information, see [Getting started with the Data Sources module](xref:Data_Sources_Setup).
 
+### 29 April - Enhancement - DataAPI 1.2.0 - Improved performance when creating new connectors
+
+The ​Data API now uses an updated version of the [Skyline.DataMiner.CICD.Models.Protocol](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Models.Protocol) NuGet package, which improves performance when creating a new connector with many standalone parameters.
+
+### 29 April - Enhancement - DataAPI 1.2.0 - Removed copyright from auto-generated connectors
+
+The auto-generated connectors generated by the Data API no longer contain the copyright message. Instead, these just contain the following line, where the timestamp indicates the UTC timestamp of when the connector was generated:
+
+```csharp
+<!--This protocol is auto-generated by the DataMiner Data API at 2024-04-12T05:43:30.823Z.-->
+```
+
+### 29 April - Fix - DataAPI 1.2.0 - All active elements updated with same identifier
+
+When a user tries to push data to the Data API, and there are stopped elements with the specified identifier, the Data API will now update all active elements with that identifier.
+
+### 29 April - New feature - DataAPI 1.2.0 - Added support for horizontal scalability of the DataAPI DxM
+
+The Data API now supports horizontal scalability: Multiple DataAPI DxMs can be installed, which will form a cluster of nodes where one node is elected as the leader and the remaining nodes are followers. Each node will broadcast a heartbeat message (every 2s), which contains the ID (GUID) of the node and an indication of whether it is the leader.
+
+During startup of a node, the node will wait for 6 seconds before determining whether the node should be the leader. After the initial startup period, the node will determine whether it should be the leader based on the heartbeat messages received from the other nodes. A node will elect itself as leader if its node ID is larger than that of any of its peers.
+
+Before the leader announces itself as leader to the other nodes (via the heartbeat message), it will retrieve all auto-generated connectors from DataMiner and keep these in memory locally.
+
+The leader knows about all auto-generated connectors, and it will coordinate all updates to these auto-generated connectors by granting a lock to any node that wishes to perform an update on an auto-generated connector.
+
+The following sequence diagram shows the flow when, for example, a follower node receives JSON data that would result in an update of a connector.
```

Comment:
This diagram is missing. Should this be added, or should this sentence be removed/rephrased?

Reply:
added diagram